app/controllers/application_controller.rb

```diff
@@ -3,5 +3,17 @@
 # :nocov:
 # Application controller
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:email, :password, :location) }
```

Comment:
I agree with both points, whenever we start to write code we should start writing test for it, in this case it would caught the missing parameter :+1: